Userland/Libraries/LibWeb/SVG/SVGTextContentElement.cpp

```diff
@@ -30,18 +30,28 @@ JS::ThrowCompletionOr<void> SVGTextContentElement::initialize(JS::Realm& realm)
     return {};
 }
 
+static float safe_parse_coordinate(DeprecatedString const& value, float fallback)
+{
+    auto coordinate = AttributeParser::parse_coordinate(value);
+    if (coordinate.has_value()) {
+        return coordinate.value();
+    }
+    dbgln("Failed to parse SVG text content coordinate value: {}", value);
+    return fallback;
+}
+
 void SVGTextContentElement::parse_attribute(DeprecatedFlyString const& name, DeprecatedString const& value)
 {
     SVGGraphicsElement::parse_attribute(name, value);
 
     if (name == SVG::AttributeNames::x) {
-        m_x = AttributeParser::parse_coordinate(value).value();
+        m_x = safe_parse_coordinate(value, m_x);
```

Comment:
looks like we can just add a check that optional has a value and that would be enough?
btw thank you for fixing that! it was very unwise to leave unconditional value() call here.

Reply:
You're welcome, but I'm not sure I understand your suggestion here - isn't `safe_parse_coordinate` doing exactly that?

Reply:
Nevermind, I think I get what you mean.

Reply:
I think something like that should be enough here:
```c++
if (maybe_value = AttributeParser::parse_coordinate(value); maybe_value.has_value())
    m_x = maybe_value.value(); 
```

Reply:
Shouldn't we also check that `name == SVG::AttributeNames::x` though? That's what bothered me.

Reply:
yes, we need to make sure that `SVG::AttributeNames::x` goes to `m_x`, `SVG::AttributeNames::x` goes to `m_y` and so on. if that's what you mean.

Reply:
You could just do: `m_x = AttributeParser::parse_coordinate(value).value_or(m_x);`

Reply:
Makes sense since we don't want the console warning, done.